### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anshuparik**
fixed spelling mistake and answers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -559,7 +559,7 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 | 403   | [Which of the following AWS Cloud services can be used to run a customer-managed relational database?](#which-of-the-following-aws-cloud-services-can-be-used-to-run-a-customer-managed-relational-database) |
 | 404   | [A user is planning to launch two additional Amazon EC2 instances to increase availability. Which action should the user take?](#a-user-is-planning-to-launch-two-additional-amazon-ec2-instances-to-increase-availability-which-action-should-the-user-take) |
 | 405   | [Which of the following can limit Amazon Storage Service (Amazon S3) bucket access to specific users?](#which-of-the-following-can-limit-amazon-storage-service-amazon-s3-bucket-access-to-specific-users) |
-| 406   | [Which AWS service allows companies to connect an Amazon VPC to an on-premises data center?](#which-aws-service-allows-companies-to-connect-an-amazon-vpc-to-an-on-premises-data-center) |
+| 406   | [Which AWS service allows companies to connect an Amazon VPC to an on-premises data center? (Select TWO)](#which-aws-service-allows-companies-to-connect-an-amazon-vpc-to-an-on-premises-data-center) |
```
Comment (danieldanielecki) [inline README.md:562]: Can you add the URL change as well? Please add `-select-two` to the end of the URL.

File: README.md [hunk 2]
```diff
@@ -2191,8 +2191,8 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 ### A customer is planning to move billions of images and videos to be stored on Amazon S3. The customer has approximately 60 Petabytes of data to move. Which of the following AWS Services is the best choice to transfer the data to AWS?
 
 - [ ] Snowball.
-- [x] S3 Transfer Acceleration.
-- [ ] Snowmobile.
+- [ ] S3 Transfer Acceleration.
+- [x] Snowmobile.
```
Comment (danieldanielecki) [inline README.md:2195]: is correct, this can stay in the PR

File: README.md [hunk 3]
```diff
@@ -4506,12 +4506,12 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
...
 **[⬆ Back to Top](#table-of-contents)**
 
-### Which AWS service allows companies to connect an Amazon VPC to an on-premises data center?
+### Which AWS service allows companies to connect an Amazon VPC to an on-premises data center? (Select Two)
 
 - [x] AWS VPN.
 - [ ] Amazon Redshift.
 - [ ] API Gateway.
-- [ ] Amazon Direct Connect.
+- [x] Amazon Direct Connect.
```
Comment (danieldanielecki) [inline README.md:4514]: if the question would be `Which AWS services or features can a company use to connect the network of its on-premises data center to AWS? (Choose two.)`, then yes, but given the question as is - it's still correct, kindly please revert this change.

File: README.md [hunk 4]
```diff
@@ -6160,7 +6160,7 @@ application needs it.
 
 **[⬆ Back to Top](#table-of-contents)**
 
-### Much AWS services provide a way to extend an on-premises architecture to the AWS Cloud? (Select TWO)
+### Which AWS services provide a way to extend an on-premises architecture to the AWS Cloud? (Select TWO)
```
Comment (danieldanielecki) [inline README.md:6163]: Please update the URL and title in the contents

**Review by danieldanielecki — COMMENTED** [review]
1 change needs to be reverted, 2 changes needs an update, 1 change stays.

File: README.md (hunk 1 above)
Comment (anshuparik) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (anshuparik) [reply README.md:6163]: done ,, by the way this question is there repeated twice

File: README.md (hunk 3 above)
Comment (anshuparik) [reply README.md:4514]: done. reverted the changes

File: README.md (hunk 4 above)
Comment (danieldanielecki) [reply README.md:6163]: Thanks, deleted the reptition in https://github.com/Ditectrev/Amazon-Web-Services-AWS-Certified-Cloud-Practitioner-CLF-C02-Practice-Tests-Exams-Questions-Answers/commit/384c032e25714ec57a7dbbacfa3e1f8e97004b9b

File: README.md [hunk 5]
```diff
@@ -5121,10 +5121,10 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 ### Which AWS tools assist with estimating costs? (Select three)
 
 - [ ] Detailed billing report.
-- [x] Cost allocation tags.
+- [ ] Cost allocation tags.
```
Comment (danieldanielecki) [inline README.md:5124]: Please revert this change.

Comment (danieldanielecki) [reply README.md:5124]: Explanation left in https://github.com/Ditectrev/Amazon-Web-Services-AWS-Certified-Cloud-Practitioner-CLF-C02-Practice-Tests-Exams-Questions-Answers/issues/58.

File: README.md [hunk 6]
```diff
@@ -5472,7 +5472,7 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 ### Which is the MINIMUM AWS Support plan that allows for one-hour target response time for support cases?
 
 - [ ] Enterprise.
-- [ ] Business.
+- [x] Business.
```
Comment (danieldanielecki) [inline README.md:5475]: Thank you, this is correct :)

File: README.md [hunk 7]
```diff
@@ -5766,8 +5766,8 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 
 - [ ] AWS Senior Support Engineers.
 - [ ] AWS Technical Account Managers.
-- [x] AWS Trusted Advisor.
-- [ ] AWS Discussion Forums.
+- [ ] AWS Trusted Advisor.
+- [x] AWS Discussion Forums.
```
Comment (danieldanielecki) [inline README.md:5770]: You are correct, thank you.

File: README.md [hunk 8]
```diff
@@ -1651,9 +1651,9 @@ We are so thankful for every contribution, which makes sure we can deliver top-n
 
 ### A company has created a solution that helps AWS customers improve their architectures on AWS. Which AWS program may support this company?
 
-- [x] APN Consulting Partners.
+- [ ] APN Consulting Partners.
 - [ ] AWS TAM.
-- [ ] APN Technology Partners.
+- [x] APN Technology Partners.
```
Comment (danieldanielecki) [inline README.md:1656]: Correct, thanks :)

**Review by danieldanielecki — COMMENTED** [review]
Approved, the one change I'll revert by myself.
